src/shared/lib/resources/affiliation.ts

```diff
@@ -31,12 +31,15 @@ export interface AffiliationSlim {
 }
 
 export interface CreateRequestBody {
-  user: string;
+  userEmail: string;
   organization: string;
   membershipType: string;
```

Comment:
This should be `membershipType: MembershipType`

Reply:
Let's talk about this on chat because it's a unique case where the back-end and front-end don't align.

Reply:
Tomorrow (wed, 2020.01.29)

Reply:
Sounds good - ping me whenever you have time.

Reply:
As discussed, the shared type should have `membershipType: MembershipType` and the back-end resource and override `membershipType` to be a `string`.

Reply:
Done! Thanks :)